appeals/web/src/server/lib/nunjucks-template-builders/gds-components-properties-jsdoc.js

```diff
@@ -0,0 +1,717 @@
+// Multipurpose Definitions (used in multiple components)
+/**
+ * @typedef TextProperty
+ * Use for Text
+ * @type {object}
+ * @prop {string} text Text to use
+ *
+ * @typedef HtmlProperty
+ * Use for HTML. Make sure to sanitise when using HTML.
+ * @type {object}
+ * @prop {string} html Text to use
+ *
+ * @typedef SharedLegendProperties
+ * @type {object}
+ * @property {string} [classes]
+ * @property {boolean} [isPageHeading]
+ *
+ * @typedef LegendProperties
+ * @type {SharedLegendProperties & TextProperty | SharedLegendProperties & HtmlProperty}
+ *
+ * @typedef SharedHintProperties
+ * @type {object}
+ * @prop {string} [id] Optional ID attribute to the span tag
+ * @prop {string} [classes] Classes to add to the span tag.
+ *
+ * @typedef SharedErrorMessageProperties
+ * - Follow the validation pattern and show an error message when there is a validation error. In the error message explain what went wrong and how to fix it.
+ * - https://design-system.service.gov.uk/components/error-message/
+ * @type {object}
+ * @prop {string} [id]	ID attribute to add to the error message span tag.
+ * @prop {string} [classes]	Classes to add to the error message span tag.
+ * @prop {string} [visuallyHiddenText] A visually hidden prefix used before the error message. Defaults to 'Error'.
+ */
+
+// Fieldset Definition
+/**
+ * @typedef FieldsetProperties
+ * - Use the fieldset component when you need to show a relationship between multiple form inputs
+ * - Documentation: https://design-system.service.gov.uk/components/fieldset/
+ * @type {object}
+ * @property {string} [describedBy] One or more element IDs to add to the aria-describedby attribute, used to provide additional descriptive information for screenreader users.
+ * @property {LegendProperties} legend Used to display text
+ * @property {string} [classes]	Classes to add to the fieldset container. https://design-system.service.gov.uk/styles
+ * @property {string} [role] Optional ARIA role attribute. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles
+ */
+
+// Accordion Definition
```

Comment:
These comments seem unnecessary (it's clear from the naming what the definition is for)